v2x/vlog_to_model.py

```diff
@@ -193,12 +204,19 @@ def vlog_to_model(infiles, includes, top, outfile=None):
 
                 # FIXME: Check if ignoring clock for "combination_sink_ports"
```

Comment:
I think this FIXME is exactly the problem you are trying to solve?

Reply:
Not exactly. I think that ignoring clock for "combination_sink_ports" for sequential logic is mandatory.

When there is an eg. flip-flop then its clock affects its output in a sequential way, therefore there should be no combinational sinks. However, when modeling a clock mux/buffer, the clock passes though it as any regular signal, so the model is sequential.

See https://docs.verilogtorouting.org/en/latest/tutorials/arch/timing_modeling/#clock-buffers-muxes, I'll add reference to that in the `README.rst`